Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointerVisual.cs

```diff
@@ -82,6 +78,11 @@ private void CheckAsset(object asset, string fieldname)
 
         public void Update()
         {
+            if (!Application.isPlaying)
```

Comment:
Why do you need this check here? I don't see why sphere pointer needs to have executealwaysattribute, and therefore why it needs this check.

Reply:
This class now uses ExecuteAlways so OnEnable is called in edit mode. Update also gets called, but because it executes more than initialization logic we exit early if we aren't in play mode.